src/tck/Reactive.Streams.TCK/packages.config

```diff
@@ -1,5 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="NUnit" version="3.2.1" targetFramework="net45" />
-  <package id="Reactive.Streams" version="1.0.1" targetFramework="net45" />
```

Comment:
Right, the dependency is added automatically during package creation with the correct version number. The TCK project references the interface now directly as project dependency but the dll isn't included in the TCK package but instead the TCK package contains a dependency to the Reactive.Streams 1.0.0.RC1 package.